packages/@webex/plugin-meetings/src/meeting/util.ts

```diff
@@ -202,7 +202,11 @@ const MeetingUtil = {
         meeting.reconnectionManager.cleanUp();
       })
       .then(() => meeting.stopKeepAlive())
-      .then(() => meeting.updateLLMConnection());
+      .then(() => {
+        if (meeting.config?.enableAutomaticLLM) {
```

Comment:
There needs to be one more UT with config object undefined.

Reply:
Added